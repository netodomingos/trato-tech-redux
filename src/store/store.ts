import { configureStore } from '@reduxjs/toolkit'
import categorysSlice from './reducers/categorias'

const store = configureStore({
   reducer: {
    categorys: categorysSlice
   } 
})

export default store