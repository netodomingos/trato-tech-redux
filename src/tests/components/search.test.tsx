import { render } from "@testing-library/react"
import { Provider } from "react-redux"

import Search from "components/search/search"
import store from "store/store"

const mockNavegacao = jest.fn()
jest.mock('react-router-dom', () => {
    return {
        useLocation: () => mockNavegacao
    }
})

describe('O componente Search', () => {
    test('Quando o componente renderiza', () => {
        const { container } = render(
            <Provider store={store}>
                <Search />
            </Provider>
        )

        expect(container).toMatchSnapshot()
    })
})