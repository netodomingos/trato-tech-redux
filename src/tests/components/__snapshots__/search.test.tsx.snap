// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`O componente Search Quando o componente renderiza 1`] = `
<div>
  <div
    class="busca"
  >
    <input
      class="input"
      placeholder="O que vocÃª procura?"
      type="text"
      value=""
    />
  </div>
</div>
`;
